Marlin/ultralcd.cpp

```diff
@@ -1403,11 +1403,18 @@ void kill_screen(const char* lcd_msg) {
     // Cooldown
     //
     bool has_heat = false;
-    HOTEND_LOOP() if (thermalManager.target_temperature[e]) { has_heat = true; break; }
-    #if HAS_TEMP_BED
-      if (thermalManager.target_temperature_bed) has_heat = true;
-    #endif
-    if (has_heat) MENU_ITEM(function, MSG_COOLDOWN, lcd_cooldown);
+    HOTEND_LOOP() if (thermalManager.target_temperature[e])
+      {
+        has_heat = true;
+        MENU_ITEM(function, MSG_COOLDOWN, lcd_cooldown);
```

Comment:
There is no point to the has_heat boolean if you're going to duplicate code for each heat test.
Also while I haven't checked yet, I'm pretty sure you're adding a Cooldown entry for every hotend.  Multi-extruder users will probably be confused